### ISSUE TITLE
Issue with UTF-8 encoding/transcoding

### ISSUE THREAD

**Issue by tmtmtmtm** [jqnatividad/qsv#772]
Given the csv file at https://gist.githubusercontent.com/tmtmtmtm/eb8e5fc3089cb70ea99544d99033999a/raw/722b57cf726a238ab392876793a1967596303996/bio.csv, and using version `qsv 0.87.1-standard-apply;fetch;foreach;generate;self_update-8-8 (aarch64-apple-darwin compiled with Rust 1.67.0) prebuilt`, `qsv dedup` fails with 

```
"bio.csv" is not UTF-8 encoded. Use the input command to transcode to UTF-8.
```

The docs for `qsv input` are a little confusing, but I _think_ they're saying that I should be able to do `qsv input bio.csv | qsv dedup`. However that fails with the same error.

I tried to reduce the source file down to a minimal test-case, but deleting almost any line from that file makes the problem go away, as does selecting any subset of columns first. Even if I select **all** the existing columns but rearrange them slightly, everything is fine: `qsv select id,name,gender,dob,dobp,dodp,dod,image bio.csv | qsv dedup`

`qsv fmt bio.csv | qsv dedup` gives the error, but `qsv fmt bio.csv --quote-always | qsv dedup` works.

`quote-always` looks like my best work-around for now, but it would be useful to know if there's a better way I should be handling files like this, or whether there's an underlying bug.

**Comment by jqnatividad** [comment]
Hi @tmtmtmtm , thanks for the report.

Just fixed this with #773.

I'll make a patch release - 0.88.1 EOD as that was a good catch and something I need for another project that uses qsv.

**Comment by tmtmtmtm** [comment]
That's great, thanks!
